.circleci/config.yml

```diff
@@ -763,7 +763,7 @@ workflows:
       - prod-jupyter-tasks
     triggers:
       - schedule:
-          cron: '0 * * * *' # run every hour
+          cron: '0 0,6,12,18 * * *' # run four times per day
           filters:
```

Comment:
Did we mean to make this change? Assuming yes but didn't see it in the description of the ticket.

Reply:
yup! should have updated the description - this was as part of convo yesterday to change from 1 hour to 4 hours